src/backend/recorders/baseRecorder.ts

```diff
@@ -85,6 +86,8 @@ export abstract class BaseRecorder<T extends WebGLObject> implements IRecorder {
     protected readonly startTime: number;
     protected readonly memoryPerSecond: { [second: number]: number };
 
+    private objects: { [id: number]: WeakRef<WebGLObject> };
```

Comment:
Should this be called `instances` instead? The code below appears to refer to individual objects as `instance`.